Sources/Transport/Streams/FoundationStream.swift

```diff
@@ -0,0 +1,107 @@
+#if os(OSX)
```

Comment:
Should be `#if !os(Linux)` or account for all other darwin systems, if someone configured this to run on iOS, they'd nedd this stream